packages/worker/src/runtime.ts

```diff
@@ -283,7 +283,7 @@ export class Runtime {
       for (const log of logs) {
         const meta: LogMetadata = {
           [LogTimestamp]: timeOfDayToBigint(log.timestamp),
-          subsystem: log.target,
+          sdkComponent: SdkComponent.core,
```

Comment:
Is the timestamp to Bigint something the bridge could do? It looks ugly to add the deprecated call with the "internal only" comment in the file above...